### PR TITLE
init: add hw_39

### PR DESCRIPTION
**Pull request by NSVoronova**

### CODE REVIEW

File: my-app/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -17,7 +19,7 @@ const Header = () => {
               ></MenuHamburger>
             <div className='search__container'><div>Search...</div><div>╳</div></div>
             <button>&#x1f50d;</button>
-            <div className='user'>Artem Malkin</div>
+            <div className='user'><img src="images/icon.png" alt="user" style={{width: "30px"}}/></div>
```
Comment (AlexKolb97) [inline my-app/src/components/Header/Header.tsx:22]: Avoid using inline styles. Create a styled img tag with styled-components. attributes are passed the same way:

<StyledImage src="images/icon.png" alr="user" />

File: my-app/src/components/Posts/BlogPage/BlogPage.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import Title from 'src/components/Title/Title'
+import PostsList from '../PostsList'
+import "./BlogPage.css"
+
+const BlogPage = () => {
+  return (
+    <>
+      <Title text='Blog'/>
+      <TabContainer/>
+
```
Comment (AlexKolb97) [inline my-app/src/components/Posts/BlogPage/BlogPage.tsx:12]: Remove spacing.

File: my-app/src/components/SignForm/SignForm.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, {ReactNode, useState} from 'react'
+import Input from './Input/Input'
+import SignButton from './SignButton/SignButton'
+import "./SignForm.css"
+
+
+const SignForm = ({ customClass, children }: { customClass: string, children: ReactNode }) => {
+  const [isSignIn, setSignIn] = useState(false);
+
```
Comment (AlexKolb97) [inline my-app/src/components/SignForm/SignForm.tsx:9]: Remove spacing.

File: my-app/src/components/SignForm/SignPage/SignPage.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <Title text="Sign In">
+        </Title>
+        <SignForm customClass="sign-in__form">
+        <Input label='E-mail' placeholder='Your E-mail' type='text' value={email} onChange={setEmail} />
+      <Input label='Password' placeholder='Your password' type='password' value={password} onChange={setPassword} />
+
+      <p className='forgot__password'>Forgot password?</p>
+      <SignButton  text={ "Sign In"} onClick = {handleClick} customClass='sign__button'/>
+        </SignForm>
+
```
Comment (AlexKolb97) [inline my-app/src/components/SignForm/SignPage/SignPage.tsx:34]: Remove spacing.

File: my-app/src/components/SignForm/SignPage/SignPage.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <>
+    <div>Back to home...</div>
+    <Title text="Sign In">
+        </Title>
+        <SignForm customClass="sign-in__form">
+        <Input label='E-mail' placeholder='Your E-mail' type='text' value={email} onChange={setEmail} />
+      <Input label='Password' placeholder='Your password' type='password' value={password} onChange={setPassword} />
+
+      <p className='forgot__password'>Forgot password?</p>
+      <SignButton  text={ "Sign In"} onClick = {handleClick} customClass='sign__button'/>
```
Comment (AlexKolb97) [inline my-app/src/components/SignForm/SignPage/SignPage.tsx:32]: You don't need the braces and spacing here. Make sure to use either single or double quotesm do not mix them. 

`<SignButton  text="Sign In" onClick={handleClick} customClass="sign__button" />`

File: my-app/src/components/SuccessfulLoginPage/SuccessfulLoginPage.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Title from '../Title/Title'
+import SignButton from '../SignForm/SignButton/SignButton'
+import "../SignForm/SignForm.css"
+import SignForm from '../SignForm/SignForm'
+
+const SuccessfulLoginPage = () => {
+  const handleButtonHomeClick =()=> {
```
Comment (AlexKolb97) [inline my-app/src/components/SuccessfulLoginPage/SuccessfulLoginPage.tsx:8]: Please pay attention to the spaces. I know it doesn't affect anything but you need to get used to "correct" code style. It's not pretty when your code style is different every time you create a function or whatever else. Please fix it :)

const handleButtonHomeClick = () => {

Comment (NSVoronova) [reply my-app/src/components/SuccessfulLoginPage/SuccessfulLoginPage.tsx:8]: I fixed all comments in PR hw_40 )

**Pull request merged**